ledger/participant-integration-api/src/main/scala/platform/indexer/parallel/ParallelIndexerFactory.scala

```diff
@@ -68,7 +68,16 @@ object ParallelIndexerFactory {
                   Executors.newFixedThreadPool(
                     1,
                     new ThreadFactoryBuilder().setNameFormat(s"ha-coordinator-%d").build,
-                  )
+                  ),
+                  throwable =>
+                    LoggingContext.newLoggingContext { implicit loggingContext =>
+                      ContextualizedLogger
+                        .get(this.getClass)
+                        .error(
+                          s"ExecutionContext ${jdbcUrl} has failed with an exception",
```

Comment:
`jdbcUrls` might contain passwords and shouldn't be logged directly.

Reply:
Removed and counting on the embedding logging context to provide enough context.